### PR TITLE
BagPart: add dropdown with max height

### LINKED ISSUE #8749: Admin - creating a item with a bagpart
### Describe the bug
The Header, in the admin area, has a z-index of 1030, so when I'm creating an item and choosing an option from a bagpart, the dropdown is appearing behind the header.
### To Reproduce
1. Go to Backoffice
2. Create an item of type with a bagpart with a lot of options
3. See the bug

### Expected behavior
The dropdown options should appear on top of everything else while choosing an option.

### Screenshots

image

image

### PR DESCRIPTION
**Pull request by agriffard**
Fixes #8749

### CODE REVIEW

**Comment by Skrypt** [comment]
I'm not sure if it's appropriate don't merge yet.

**Comment by agriffard** [comment]
It is the same css classes used for the 'Add widget' in the form part.

/cc @ns8482e

**Comment by agriffard** [comment]
@Skrypt Do you see another way to fix it?

**Comment by Skrypt** [comment]
Looks ok but why the scrollbar background is transparent?

**Comment by Skrypt** [comment]
Also ::-webkit-scrollbar only works in Chromium based browsers.

File: src/OrchardCore.Modules/OrchardCore.Flows/Assets/scss/flows.edit.scss [hunk 1]
```diff
@@ -20,15 +20,15 @@
     } 
 }
 
-.btn-widget-add-below, .btn-widget-metadata {
+.btn-widget-add-below, .btn-widget-metadata, .btn-bag-add {
     ::-webkit-scrollbar {
         width: 4px;
```
Comment (Skrypt) [inline src/OrchardCore.Modules/OrchardCore.Flows/Assets/scss/flows.edit.scss:25]:
```suggestion
        width: 4px;
        background-color: white;
```

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.Flows/Assets/scss/flows.edit.scss:25]: Normally this color should be configurable.

**Review by Skrypt — APPROVED** [review]

**Pull request merged**